src/compositions/useLink.ts

```diff
@@ -110,6 +119,8 @@ export function useLink(
 
   const isMatch = computed(() => isRoute(router.route) && router.route.matches.some((match) => match.id === route.value?.id))
   const isExactMatch = computed(() => router.route.id === route.value?.id)
+  const isActive = computed(() => isRoute(router.route) && isDefined(route.value) && router.route.href.startsWith(route.value.href))
+  const isExactActive = computed(() => router.route.href === route.value?.href)
```

Comment:
I think I'd be tempted to use `isMatch` and `isExactMatch` inside of `isActive`. I think of these as MORE narrow guards on top of those and without using them inside of the function it could lead to surprises like a route that uses something like `/[userId]` being discovered as `isActive` for the actual route of `/home`. That's probably not the best example, but I think you get the point

Reply:
Yeah it's technically possible to have a url that matches multiple routes. But the first route will always match based on the order they're defined.

I also don't think the route matters for whether the route is active or not. And it be weird I think if a link exactly matches the current url but wasn't marked as "exact active".